pkg/config/mesh/networks_watcher.go

```diff
@@ -64,7 +64,7 @@ func NewNetworksWatcher(fileWatcher filewatcher.FileWatcher, filename string) (N
 	}
 
 	ResolveHostsInNetworksConfig(meshNetworks)
-	networksdump, _ := (&jsonpb.Marshaler{}).MarshalToString(meshNetworks)
+	networksdump, _ := gogoprotomarshal.ToJSONWithIndent(meshNetworks, "   ")
```

Comment:
nit: I think you're indent is inconsistent. Maybe just use 2 spaces for everything?